.nycrc

```diff
@@ -10,6 +10,7 @@
 	"exclude": [
 		"coverage",
 		"test",
-		"tests"
+		"tests",
+		"lib"
```

Comment:
This was necessary because it takes a long time to load eslint-plugin-import.

https://github.com/import-js/eslint-plugin-import/pull/2191#discussion_r690004072